app/models/user.rb

```diff
@@ -34,14 +25,11 @@ class User < ApplicationRecord
                             unless: ->(u) { u.test_server? }
   after_initialize :set_display_name, :set_time_zone
 
-  validates_format_of       :type,          with: /\AStudent|Admin|Ta|TestServer\z/
   validates_inclusion_of    :locale, in: I18n.available_locales.map(&:to_s)
 
   # role constants
-  STUDENT = 'Student'
-  ADMIN = 'Admin'
-  TA = 'Ta'
-  TEST_SERVER = 'TestServer'
+  STANDARD = 'Human'.freeze
```

Comment:
remember to make a `Human` class as well 
rename the `STANDARD` variable to `HUMAN`